metagraph/src/cli/config/config.hpp

```diff
@@ -83,10 +83,13 @@ class Config {
     unsigned int port = 5555;
     unsigned int bloom_max_num_hash_functions = 10;
     unsigned int num_columns_cached = 10;
+    unsigned int max_hull_forks = 0;
 
     unsigned long long int query_batch_size_in_bytes = 100'000'000;
     unsigned long long int num_rows_subsampled = 1'000'000;
     unsigned long long int num_singleton_kmers = 0;
+    unsigned long long int max_hull_depth
+        = std::numeric_limits<unsigned long long int>::max();
```

Comment:
By default, the `max_hull_depth` will be restricted to 2 * read_length because of the other parameters. I can change `max_hull_forks` to 4 by default.